src/routes/transactions/mappers/common/transaction-info.mapper.ts

```diff
@@ -15,12 +15,15 @@ import { NativeCoinTransferMapper } from './native-coin-transfer.mapper';
 import { SettingsChangeMapper } from './settings-change.mapper';
 import { DataDecoded } from '../../../data-decode/entities/data-decoded.entity';
 import { DataDecodedParameter } from '../../../data-decode/entities/data-decoded-parameter.entity';
+import { HumanDescriptionMapper } from '../common/human-description.mapper';
+import { IConfigurationService } from '../../../../config/configuration.service.interface';
 
 @Injectable()
 export class MultisigTransactionInfoMapper {
   private readonly TRANSFER_METHOD = 'transfer';
   private readonly TRANSFER_FROM_METHOD = 'transferFrom';
   private readonly SAFE_TRANSFER_FROM_METHOD = 'safeTransferFrom';
+  private readonly HUMAN_DESCRIPTION_ENABLED: boolean | undefined;
```

Comment:
I would rename this to something like:

```typescript
isHumanDescriptionFeatureEnabled
```

or

```typescript
isHumanDescriptionEnabled
```

The reason is that from a style perspective all-caps would mean a global variable known at build time.

Even though we don't strictly follow this, it's better explained here: https://google.github.io/styleguide/tsguide.html#identifiers